### DISCUSSION TITLE
Commits from github-actions[bot] are not counted

### DISCUSSION THREAD

**Discussion by JayantGoel001** [lowlighter/metrics#134] [Q&A] [answered]
Hey @lowlighter , 
I am observing these days that GitHub actions generated by metrics are not being counted to my commits although I see them in my repo but not in whole contributions.
Any Idea About It?

**Comment by lowlighter** [comment, marked as answer]
Hello !

Sorry, I'm not sure to understand fully your question, so I'm not the answer below is what you asked 😕.

If you're talking about why metrics' commits aren't counted towards your total commits count, that's because `GITHUB_TOKEN` use a special account named `github-actions` which acts on your behalf, without being totally realated to your account. You can check this behaviour in your [contributions page](https://github.com/JayantGoel001/JayantGoel001/graphs/contributors) (you currently have ~260 commits while your repo has ~580 commits).

In firsts releases, metrics used the same token for both fetching data and committing, so it counted as a commit for token owner. 

___

If that wasn't what you ask, could you elaborate a bit more 🙂  ?

Thanks !

**Comment by JayantGoel001** [comment]
Yes , I was asking same thing now I get it. 😁
Thank you 😃
